icu4c/source/i18n/unicode/numberformatter.h

```diff
@@ -2493,6 +2541,14 @@ class U_I18N_API FormattedNumber : public UMemory, public FormattedValue {
     /** @copydoc FormattedValue::nextPosition() */
     UBool nextPosition(ConstrainedFieldPosition& cfpos, UErrorCode& status) const U_OVERRIDE;
 
+    /**
+     * Gets the resolved output unit.
+     *
+     * @return `MeasureUnit`.
+     * @draft ICU 67
+     */
+    MeasureUnit getOutputUnit(UErrorCode& status) const;
```

Comment:
Please put this after `toDecimalNumber` and put it inside of U_HIDE_DRAFT_API blocks